pkg/blobstore/configuration/blob_replicator_creator.go

```diff
@@ -16,4 +16,8 @@ type BlobReplicatorCreator interface {
 	// type. For example, sending replication requests over gRPC is
 	// only supported for the Content Addressable Storage.
 	NewCustomBlobReplicator(configuration *pb.BlobReplicatorConfiguration, source blobstore.BlobAccess, sink BlobAccessInfo) (replication.BlobReplicator, error)
+
+	// GetStorageTypeName() returns the name of the storage type that
+	// this BlobReplicatorCreator is able to create BlobReplicators for.
+	GetStorageTypeName() string
```

Comment:
Note that BlobAccessCreator inherits BlobReplicatorCreator. Please remove GetStorageTypeName() from the latter, and also remove the implementations from {cas,ac,...}BlobAccessCreator.

Reply:
I see, I removed it from BlobAccessCreator, but I don't think I can remove it from the {cas,ac,...}BlobAccessCreators? since they don't inherit from the respective replicators, and it still needs the implementation to make it a legit BlobAccessCreators?

Reply:
nvm, i did see that it inherits the replicator, removing it.